### ISSUE TITLE
Processing manual-ramp-inspektora fails due null pointer exception

### ISSUE THREAD

**Issue by blcham** [kbss-cvut/bpmn2stamp#56]
Command:
```
cd bpmn-convertor-examples/processes/ucl
./bin/process-bpmn.sh bpmn/manual-ramp-inspektora.bpmn
```

Returns error:
```
Converting BBO file 'manual-ramp-inspektora-bpmn.ttl' to STAMP file 'manual-ramp-inspektora-pre-stamp.ttl' 
Exception in thread "main" java.lang.NullPointerException 
       at cz.cvut.kbss.bpmn2stamp.converter.mapper.bbo2stamp.MapstructBbo2StampMapper.lambda$processControlledProcessProperties$0(MapstructBbo2StampMapper.java:56) 
       at cz.cvut.kbss.bpmn2stamp.converter.mapper.bpmn2bbo.MapstructBpmn2BboMapper$AfterMappingAction.lambda$new$0(MapstructBpmn2BboMapper.java:419) 
```

Full log: 
[manual-ramp-inspektora.log](https://github.com/kbss-cvut/bpmn2stamp/files/11174449/manual-ramp-inspektora.log)

**Comment by grigobog** [comment]
The problem was caused by the wrong conversion of the sequence flows, having unconvertable parts as a source or target reference. In this particular case it was caused by the gateways.
The fix will be implemented in the issue #52.

**Comment by blcham** [comment]
tested and works.
